src/libs/addEmojiToComposerTextInput/addEmojiToComposerTextInput.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
> On a side note, it seems this logic can add an emoji to any `textInput`

Unfortunately it can't, as it uses the `setTextAndSelection` method, which is only available on the composer component 😅 Otherwise agreeing!

Reply:
I renamed it to `addEmojiToComposer`, makes more sense, as you can only add to the composer

Reply:
Oh, coming back to this, could we implement `setTextAndSelection` directly in the `RNTextInput`?